src/docs/admin/references/NotificationServiceReference.adoc

```diff
@@ -96,35 +130,73 @@ if not provided the extension of the file will determine the appropriate engine
 notification.script.engine=bash
 ----
 
-Finally, multiple sequential script execution can be enabled.
-Users must have an active subscription for an event and the Third party notification method activated.
-It is set to false by default.
+[[execution-mode]]
+===== Execution modes
+====== Single execution mode
+
+By default Third party notification method is configured in single execution mode.
+In single execution mode, only one script execution will occur for a given notification
+regardless of how many users are notified.
 
 For example if 5 users are subscribed to the Scheduler Frozen event with the Third party notification
-method active and multiple execution enabled, then each time the  <<_glossary_proactive_scheduler,*ProActive Scheduler*>> freezes
-the script will be executed one time per notified user so in this example 5 times.
-If it is set to false, only one execution will be performed regardless of how many users are notified.
+method active then only one script execution will take place.
 
 ----
 #if false execute the script one time per event even if there are several subscribers
 #if true execute the script for each subscribers for each event
 notification.script.multiple.execution=false
 ----
 
-====== Reading notification data
+[[multiple-executions-mode]]
+====== Multiple executions mode
+
+To enable the multiple execution mode and access every notified users parameter inside the script simply change to property to true.
```

Comment:
To enable the multiple execution mode and access every notified users parameter inside the script, simply change the property to true.